### ISSUE TITLE
How might we create a plain language summary for current and future conditions?

### ISSUE THREAD

**Issue by colinmurphy01** [18F/weather.gov#69]
### Description
Plain language summaries of the the near-term forecast are valuable to all users (based on user interviews), especially for replacing technical weather jargon (apparent temp = "feels like") and accessibility needs. 

We found that a 24-hour outlook is valuable for partners in planning for potential hazards (user interview). 

**Related Boulder: #57 **

### High level takeaways: 

- When used in a consistent and purposeful way, weather narratives deliver unique value 
- For prototyping, a weather narrative is a text summary of the forecast for the next few days, 200 words max
- They should be simple and easy for non-weather experts to understand 
- We have some high-level guidance for what these look like, all of which is flexible, especially when hazardous weather is present or on its way

[Full findings and initial recommendations documented here](https://docs.google.com/document/d/1HlRUiS9RBywIaceAC_kVO_NSf8xol19ctxai0q4_0Nc/edit?usp=sharing)

### Suggested component
Text narrative component that a forecaster can use to publish a weather narrative to the website
Component details TBD (need clarity on next steps)

### Stakeholder research
(all questions for both regular conditions and weather hazards): 
- [ ]  How do mets understand the differences between weather stories, discussion synopses, discussions, and DSS resources? 
- [ ] Who are the intended audiences for each of these?
- [ ] What is the specific purpose for each?
- [ ] What goes into creating them? 
- [ ] Which formats are mets most/least comfortable creating? Why? 

### User research (to test later)
- [ ] What is the optimal length of a text narrative?
- [ ] Is optimal length different on a regular vs hazardous weather day?
- [ ] What information do they expect NWS to provide? 
- [ ] What information do they not expect NWS to provide? 
- [ ] Bonus: Would video be as/more effective? 

### Out of scope
- [ ] How do other sites communicate risk levels, hazard descriptions, actions, etc (look at [ready.gov](http://ready.gov/), state sites, etc)
- [ ] What tools/resources might we consider providing to mets to help them create weather narratives? 

```[tasklist]
### Pebbles
- [x] Conduct desk/background research
- [x] Document initial recommendations
- [x] Determine if there is enough information to start design in parallel
- [x] update/create new user stories for design
```

**Comment by colinmurphy01** [comment]
Read through the findings and recommendations. Looks good to me and ready to be prototyped. 

Can you update the ticket to reflect the next phase of work? Also feel free to migrate the research questions into the doc to simplify this ticket.

**Comment by kmranjo** [comment]
@colinmurphy01  We got some answers for the "stakeholder research" questions during the IRL last week, but given the differences we see office to office and region to region, I'm wondering if it makes sense to take those as the only answers. Do you think we need to expand that to include a larger sample size? @sarahmhgov would love your thoughts on this too.

**Comment by colinmurphy01** [comment]
I don't think they should be perceived as the only answers.

We will try to have more WFO onsite visits across our team's footprint. So we could consider adding them to our onsite agenda?

However because of the discrepancies between offices I don't think it should block us from bringing a recommendation to the table and getting feedback.

**Comment by kmranjo** [comment]
Removed the outdated tasks on here, as they don't really apply anymore (they were specific to the IRL, but from before we had fully planned it). I think next steps for this beyond visual design would be to gut-check the solution with eng and with forecasters for feasibility.

**Comment by colinmurphy01** [comment]
We've aligned on a path forward for #110 from a design and product perspective. This issue can be closed.
